easybuild/easyconfigs/l/libpng/libpng-1.6.16-CrayGNU-2015.11-XC.eb

```diff
@@ -0,0 +1,19 @@
+easyblock = 'ConfigureMake'
+
+name = 'libpng'
+version = '1.6.16'
+
+homepage = 'http://www.libpng.org/pub/png/libpng.html'
+description = "libpng is the official PNG reference library"
+
+toolchain = {'name': 'CrayGNU', 'version': '2015.11-XC'}
+toolchainopts = {'dynamic': True, 'pic': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [('zlib', '1.2.8')]
+
+configopts = "--disable-static"
```

Comment:
Same explanation as before, we didn't use CRAYPE_LINK_TYPE but needed the wrapper to do the right thing or to disable static building in the case of zlib.